### DISCUSSION TITLE
Support for dynamic tables

### DISCUSSION THREAD

**Discussion by cng-apelletier** [Snowflake-Labs/terraform-provider-snowflake#2019] [Ideas]
I've started to play around with dynamic tables in Snowflake and, while it is still in feature preview, I can't seem to find any mention of it on this repo.

Are there plans to add it soon? I'd really like to be able to manage permissions for these new objects from my terraform project.

**Comment by melaraj2** [comment]
Same here.  We need support for Dynamic tables ASP.  Here is an issue i am tracking #1922
